### PR TITLE
add module invoice_line_sale_link that creates links between parent line...

### PR DESCRIPTION
**Pull request by chafique-delli**
...s orders and child in invoice line for bundle product.

This Pull Request depends of the pull request OCA/stock-logistics-workflow#101

### CODE REVIEW

File: invoice_line_sale_link/account_invoice.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    _inherit = 'account.invoice.line'
+    _order = 'sale_parent_line_id, name desc'
+
+    def _get_invoice_line_from_order_line(self, cr, uid, ids, context=None):
+        invoice_line_ids = self.pool['account.invoice.line'].search(
+            cr, uid, [('sale_line_id', 'in', ids)], context=context)
+        return invoice_line_ids
+
+    _columns = {
+        'sale_line_id': fields.many2one(
```
Comment (eLBati) [inline invoice_line_sale_link/account_invoice.py:35]: So, this https://github.com/OCA/sale-workflow/pull/139#discussion_r28220127 is still valid

**Comment by chafique-delli** [comment]
Runbot test failed because it try to install module 'invoice_line_sale_link' that depends on module 'picking_line_sale_link'.But the latter module is not available in this repository.

File: invoice_line_sale_link/account_invoice.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            'Sales Order Line',
+            ondelete='set null',
+            select=True,
+            readonly=True),
+        'sale_parent_line_id': fields.related(
+            'sale_line_id',
+            'line_parent_id',
+            type='many2one',
+            relation='sale.order.line',
+            string='Parent Product',
```
Comment (guewen) [inline invoice_line_sale_link/account_invoice.py]: Parent Sale Line?

**Comment by pedrobaeza** [comment]
Closing as inactivity, conflicts and red status. If you are still interested in this, feel free to open a new PR.

**Pull request closed without merging**